src/main/java/io/confluent/connect/hdfs/HdfsSinkConnectorConfig.java

```diff
@@ -301,6 +302,7 @@ protected HdfsSinkConnectorConfig(ConfigDef configDef, Map<String, String> props
     partitionerConfig = new PartitionerConfig(partitionerConfigDef, originalsStrings());
     this.name = parseName(originalsStrings());
     this.hadoopConfig = new Configuration();
+    this.url = extractUrl(originalsStrings());
```

Comment:
why do we need to make a copy here? just use props directly?

Reply:
following the general pattern as above but I will change